app/javascript/src/components/Common/Sidebar/index.jsx

```diff
@@ -48,17 +49,19 @@ const Sidebar = () => {
     <NeetoUISidebar
       appName={APP_NAME}
       changelogProps={{ id: "neetochangelog-trigger" }}
+      isCollapsed={isSidebarCollapsed}
       navLinks={SIDENAV_LINKS}
       organizationInfo={{
         name: "Wheel",
         subdomain: "bigbinary.com",
       }}
       profileInfo={{
         name: `${user.first_name} ${user.last_name}`,
-        imageUrl: user.profile_image_path,
+        imageUrl: "https://i.pravatar.cc/300",
         email: user.email,
         bottomLinks,
       }}
+      onCollapse={() => setIsSidebarCollapsed(!isSidebarCollapsed)}
```

Comment:
https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update